pages/agents/[agentId]/index.tsx

```diff
@@ -191,8 +208,8 @@ export default function AgentPage() {
 
   return (
     <Box
-      component="main"
-      className="MainContent"
+      component='main'
```

Comment:
There's an issue with your code formatter, double quote should not be changed to single!

Reply:
I'll draft this, n make a clean PR